projects/igniteui-angular/src/lib/slider/thumb/thumb-slider.component.ts

```diff
@@ -78,6 +78,16 @@ export class IgxSliderThumbComponent implements OnInit, OnDestroy {
     @HostBinding('attr.z-index')
     public zIndex = 0;
 
+    @HostBinding('attr.aria-valuenow')
+    public get ariaValueNow() {
+        return this.value;
+    }
+
+    @HostBinding('attr.aria-labelledby')
```

Comment:
`aria-labelledby` is expected to contain a refreence to a label element, isnt it? I assume we need to se just aria-label attr? Please check that